modules/mintmaker/pages/rpm-lockfile.adoc

```diff
@@ -29,6 +29,18 @@ Please find how to generate RPM lockfile in xref:ROOT:building:prefetching-depen
 
 MintMaker searches the repository for files named `rpms.in.yaml`/`rpms.in.yml` and assumes that a lockfile named `rpms.lock.yaml`/`rpms.lock.yml` exists in that same directory. MintMaker supports multiple lockfiles in a repository, but they must be placed in different directories.
 
+== How to enable automerge for RPM security updates
+
+It's possible to configure RPM security updates to automerge based on the severity of the fixed CVEs. To enable this functionality, add the following setting to your renovate configuration:
+
+[source,json]
+----
+{
+  "rpmVulnerabilityAutomerge": "ALL" | "MEDIUM" | "HIGH" | "CRITICAL"
```

Comment:
Is it possible to change this to support list of severities? For example users may want to auto-merge high and critical CVE fixes.